build-system/tasks/helpers.js

```diff
@@ -367,6 +367,24 @@ async function esbuildCompile(srcDir, srcFilename, destDir, options) {
       fs.outputFile(`${destFile}.map`, map),
     ]);
 
+    // TODO(alanorozco): Sourcemaps are probably very broken.
+    if (options.outputFormat === 'amd') {
```

Comment:
Is another full esbuild process necessary here? I think we can just process via babel to transform from ESM into AMD.

Reply:
We can do just babel.